README.md

```diff
@@ -1,13 +1,38 @@
 # Eppo SDK for Golang
 
-## Building
+Eppoclient is a client sdk for `eppo.cloud` randomization API.
+It is used to retrieve the experiments data and put it to in-memory cache, and then get assignments information.
 
-```
-make build
-```
+## Getting Started
+
+Refer to our [SDK documentation](https://docs.geteppo.com/feature-flagging/randomization-sdk/) for how to install and use the SDK.
```

Comment:
Docs were moved to a new link: https://docs.geteppo.com/prerequisites/feature-flagging/randomization-sdk/

Reply:
Changed